jenkins/bootstrap_test.py

```diff
@@ -2296,6 +2296,8 @@ def test_valid_job_envs(self):
                     if (
                             '--env-file=jobs/platform/gce.env' not in args
                             and '--env-file=jobs/platform/gke.env' not in args
+                            and '--env-file=jobs/platform/gke-staging.env' not in args
+                            and '--env-file=jobs/platform/gke-prod.env' not in args
```

Comment:
I flipped a couple to only reference one of them (now that `gke.env` is just an endpoint selection).